packages/sources/css/src/components/text-input/src/index.css

```
@@ -52,19 +52,31 @@ textarea.vtmn-text-input {
 
 .vtmn-text-input[disabled] {
   cursor: not-allowed;
-  background-color: var(--vtmn-color_grey-light-3);
-  border-color: var(--vtmn-color_grey-light-1);
-  color: var(--vtmn-color_grey-light-1);
-}
-
-.vtmn-text-input[disabled]::placeholder {
-  color: var(--vtmn-color_grey-light-1);
+  opacity: 0.38;
 }
 
 .vtmn-text-input_label {
   color: var(--vtmn-color_black);
   font-size: rem(16px);
   line-height: 1;
+  display: block;
+}
+
+.vtmn-text-input_container {
+  position: relative;
+  display: inline-block;
+}
+
+.vtmn-text-input + span {
```

What about use class selector ?
```suggestion
.vtmn-text-input + .vtmn-text-input_icon {
```

---

Thanks for the suggestion